android/build.gradle

```diff
@@ -73,10 +78,9 @@ tasks.register("sourcesJar", Jar) {
 publishing {
     publications {
         maven {
-            from components.android
-
-            artifact javadocJar
-            artifact sourcesJar
+            afterEvaluate {
```

Comment:
Is the afterEvaluate necessary? Ugh, [apparently it is](https://developer.android.com/build/publish-library/upload-library#create-pub).